docs/FastAPI.md

```diff
@@ -64,6 +64,17 @@ This information is included in the response from the `/__about` endpoint.
 
 The `version` information is included in the response from the `/__about` endpoint.
 
+### Security
+
+* `SECURITY_ENABLED` - OPTIONAL, DEFAULT=False, Exposed to the UI - Indicates if security is enabled.
+* `SECURITY_CLIENT_ID` - REQUIRED iff `SECURITY_ENABLED=True`, Exposed to the UI - The registered application (client) ID.
```

Comment:
iff -> if

Reply:
`iff` is short for `if and only if`. If you prefer I can write it as `if and only if`